### ISSUE TITLE
Task watchdog got triggered.

### ISSUE THREAD

**Issue by 2200913** [atomic14/voice-controlled-robot#17]
Hi, i am trying to run this on a esp32 and am getting a few issues.

Most of the time i get this error:

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4
Starting up
Loading model
12 bytes lost due to alignment. To avoid this loss, please make sure the tensor_arena is 16 bytes aligned.
Used bytes 22644

Output Size 1

Created Neral Net
m_pooled_energy_size=43
Created audio processor
Starting i2s
E (22481) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
E (22481) task_wdt:  - IDLE (CPU 0)
E (22481) task_wdt: Tasks currently running:
E (22481) task_wdt: CPU 0: Command Detect
E (22481) task_wdt: CPU 1: IDLE
E (22481) task_wdt: Print CPU 0 (current core) backtrace

Backtrace:0x400DEE5F:0x3FFD2D9C |<-CORRUPTED

E (22481) task_wdt: Print CPU 1 backtrace

Backtrace:0x4008360D:0x3FFD339C |<-CORRUPTED

Average detection time 111ms

Sometimes it doesnt give that error, but it doesnt detect any of the keywords as well.
Please help, i am very interested on this project

**Comment by sarasqz** [comment]
Hello, I've been facing the same issues as you. Did you manage to solve it?

**Comment by siva-dharshan** [comment]
Hello, I am having the same issue how to solve it?

E (22462) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
E (22462) task_wdt:  - IDLE (CPU 0)
E (22462) task_wdt: Tasks currently running:
E (22462) task_wdt: CPU 0: Command Detect
E (22462) task_wdt: CPU 1: IDLE
E (22462) task_wdt: Print CPU 0 (current core) backtrace


Backtrace: 0x400DF44F:0x3FFD2FCC |<-CORRUPTED

  #0  0x400DF44F:0x3FFD2FCC in task_wdt_isr at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_system/task_wdt.c:183 (discriminator 3)

E (22462) task_wdt: Print CPU 1 backtrace


Backtrace: 0x4008367D:0x3FFD37FC |<-CORRUPTED

  #0  0x4008367D:0x3FFD37FC in esp_crosscore_isr at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_system/crosscore_int.c:92

Average detection time 111ms

**Comment by aegorgor** [comment]
Has anyone managed to resolve this issue? Not sure if it is related to it but I am getting prediction number 4 constantly which is background noise.

**Comment by salvatore1907** [comment]
still getting this problem. any solutions?

**Comment by aegorgor** [comment]
sorry its been a while so i dont really remember. I believe I had to revert to an earlier version of the library (think it was the esp library)?

**Comment by salvatore1907** [comment]
> sorry its been a while so i dont really remember. I believe I had to revert to an earlier version of the library (think it was the esp library)?

yes, i’ve downgraded version to 3.x.x, code worked but now i’ve problem with microphone.
thanks.

**Comment by Amir20241403** [comment]
> > sorry its been a while so i dont really remember. I believe I had to revert to an earlier version of the library (think it was the esp library)?
> 
> yes, i’ve downgraded version to 3.x.x, code worked but now i’ve problem with microphone. thanks.

hello.witch library you downgraded version and how downgrad that library in VSCode?

**Comment by salvatore1907** [comment]
> > > sorry its been a while so i dont really remember. I believe I had to revert to an earlier version of the library (think it was the esp library)?
> > 
> > 
> > yes, i’ve downgraded version to 3.x.x, code worked but now i’ve problem with microphone. thanks.
> 
> hello.witch library you downgraded version and how downgrad that library in VSCode?

check https://github.com/atomic14/voice-controlled-robot/issues/22

**Comment by Ahd1010** [comment]
Hello All , I got the same error (E (22462) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
E (22462) task_wdt: - IDLE (CPU 0)
E (22462) task_wdt: Tasks currently running:
E (22462) task_wdt: CPU 0: Command Detect
E (22462) task_wdt: CPU 1: IDLE
E (22462) task_wdt: Print CPU 0 (current core) backtrace

Backtrace: 0x400DF44F:0x3FFD2FCC |<-CORRUPTED) Any one please can guide how to solve this error , I am really interested in this project ,Thnak you so much .
